CONTRIBUTING.md

```diff
@@ -1,16 +1,518 @@
-# Contributing
+# Contributing to the Conjur Google Cloud Marketplace Application
 
-For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
+Thank you for your interest in the Conjur Google Cloud Marketplace Application. Before contributing, please take a moment to read and sign our <a href="https://github.com/cyberark/community/blob/master/documents/CyberArk_Open_Source_Contributor_Agreement.pdf" download="secretless-broker_contributor_agreement">Contributor Agreement</a>.
+This provides patent protection for all Secretless Broker users and allows CyberArk to enforce its license terms. Please email a signed copy to
+<a href="oss@cyberark.com">oss@cyberark.com</a>.
```

Comment:
These links should use the markdown standard (`[name](url)`) instad of the html `<a />` anchors